pushsync/simulation_test.go

```diff
@@ -56,7 +56,7 @@ var (
 
 	nodeCntFlag   = flag.Int("nodes", 4, "number of nodes in simulation")
 	chunkCntFlag  = flag.Int("chunks", defaultChunkCnt, "number of chunks per upload in simulation")
-	testCasesFlag = flag.Int("cases", 4, "number of concurrent upload-download cases to test in simulation")
+	testCasesFlag = flag.Int("cases", 2, "number of concurrent upload-download cases to test in simulation")
```

Comment:
Even though we added a limit of 32 concurrent get requests, we still concurrently run all the test cases, which appears to be too many for Travis.